packages/core/src/browser/style/tree.css

```diff
@@ -171,10 +171,6 @@
     opacity: var(--theia-mod-disabled-opacity);
 }
 
-.theia-tree-element-node {
-    width: 100%
-}
```

Comment:
I'm not sure if there is any regression with trees with this change since it is affects all trees but I haven't encountered any.

Reply:
I believe this class in particular is exclusively used in the [`SourceTreeWidget`](https://github.com/eclipse-theia/theia/blob/1dcddc630ab0beb5d66ec885d207d3e6529fb692/packages/core/src/browser/source-tree/source-tree-widget.tsx#L89), which is only used by the expression watching stuff (i.e. the `watch` debug tab and the debug console). In both cases it didn't do anything other than accidentally overflowing the tree node (see picture above).